BraveRewardsUI/Wallet/WalletViewController.swift

```diff
@@ -156,17 +157,29 @@ class WalletViewController: UIViewController, RewardsSummaryProtocol {
       guard let host = state.url.host else { return }
       attentionView.valueLabel.text = "0%"
       
-      state.ledger.publisherInfo(forId: host) { info in
-        guard let publisher = info else { return }
-        assert(Thread.isMainThread)
-        publisherView.setVerified(publisher.verified)
+      publisherView.onCheckAgainTapped = { [weak self] in
+        guard let self = self else { return }
         
-        if let percent = self.state.ledger.currentActivityInfo(withPublisherId: publisher.id)?.percent {
-          attentionView.valueLabel.text = "\(percent)%"
+        publisherView.checkAgainButton.isLoading = true
+        self.state.ledger.publisherInfo(forId: host) { info in
+          publisherView.checkAgainButton.isLoading = false
+          
+          guard let publisher = info else { return }
+          assert(Thread.isMainThread)
+          publisherView.setVerified(publisher.verified)
+          
+          if publisher.verified {
+            publisherView.checkAgainButton.isHidden = true
```

Comment:
This should hide regardless of if the refreshed publisher is verified

Reply:
Fixed.